datadog/README.MD

```diff
@@ -0,0 +1,56 @@
+# Kubecost with Datadog integration
+
+## Overview
+
+This repository is designed to give you a solution to integrate Kubecost with Datadog. By default, Kubecost will emit cost allocation metrics (unaggregrated, unreconciled) in a Prometheus compatible format with cloud providers) via `/metrics` endpoint. This solution installs Kubecost with additional annotations in cost-model pods, which allows Datadog agent to look into the `/metrics` endpoint, map prometheus compatible metrics to the Datadog compatible metrics, then push the metrics to your Datadog account. 
+
```

Comment:
maybe add caveat section?
https://github.com/kubecost/cost-analyzer-helm-chart/tree/develop/cost-analyzer/grafana-templates#caveats

Reply:
Great feedback ! Thank you, I added it into README in the latest commit.